repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestProjectUtils.scala

```diff
@@ -9,7 +9,7 @@ object TestProjectUtils {
   val projectUuid: String = UUID.randomUUID().toString
 
   def getDummyProject(
-    projectId: ProjectId = ProjectId(projectUuid),
+    projectId: ProjectId = ProjectId(UUID.randomUUID().toString),
```

Comment:
Unfortunately, now there is a duplication of logic in how a `ProjectId` is created for tests. It would be nice if you will find a way to fix it.

Reply:
fixed

Reply:
You made it even worse :) Now the logic is duplicated everywhere. We should avoid such things.

Reply:
Then I have no idea how to fix this, because for my test's I need non-constant ProjectId. I can get non-static ProjectId if I will create him in getDummyProject(), not in object. But, unfortunately, this is duplication of logic. I suggest in tests ,that need different ProjectId, create it and pass it to the parameters explicitly, as I had from the very beginning.

Reply:
```
object TestProjectUtils {

  def getDummyProjectId: ProjectId = ProjectId(randomUuidStr)

  def getDummyProject(
    projectId: ProjectId = getDummyProjectId,
    ownerId: UserId = getDummyUserId,
    name: String = s"project-$randomUuidStr",
    repository: String = s"repo-$randomUuidStr",
    active: Boolean = true
  ): Project =
    Project(projectId, ownerId, name, repository, active)

  // todo move this method to the TestUserUtils
  private def getDummyUserId: UserId = UserId(randomUuidStr)

  private def randomUuidStr: String = UUID.randomUUID.toString
}
}
```
Notice that fields `name` and `repository` contain different `UUID`s than `projectId`. This is okay since `name` and `repository` can be arbitrary and may not contain `projectId` in general. They must be unique and that's the only requirement for them at the moment.
The above code also introduces the method `getDummyProjectId` that will allow anyone to generate `projectId` without knowing how to do this.

Reply:
Got it.